src/app/CASESessionManager.cpp

```diff
@@ -188,6 +188,12 @@ Optional<SessionHandle> CASESessionManager::FindExistingSession(const ScopedNode
         peerId, MakeOptional(Transport::SecureSession::Type::kCASE), transportPayloadCapability);
 }
 
+void CASESessionManager::ReleaseSession(const ScopedNodeId & peerId)
+{
+    auto * session = mConfig.sessionSetupPool->FindSessionSetup(peerId, false);
+    ReleaseSession(session);
```

Comment:
Doesn't this need to check whether session is null?

Reply:
ReleaseSession is doing a null check already. It is 2 lines under this method and this is why I have not added one here

Reply:
Ah, OK.